test/unit/build/services/github_test.js

```diff
@@ -33,13 +42,22 @@ describe('#getPullRequests', function() {
 
   it('should call the Github API with the label without space', async function() {
     // given
-    const expectedUrl = 'https://api.github.com/search/issues?q=is:pr+is:open+archived:false+sort:updated-desc+user:1024pix+label:Tech%20Review%20Needed';
+    const items = [
+      { html_url: 'http://test1.fr', number: 0, title: 'PR1', labels: [ { name: 'team certif'} ]},
+    ];
+    nock('https://api.github.com')
+        .get(`/search/issues?q=is%3Apr+is%3Aopen+archived%3Afalse+user%3Agithub-owner+label%3ATech%2520Review%2520Needed&sort=updated&order=desc`)
+        .reply(200, {items})
+
+    nock('https://api.github.com')
+        .get('/repos/github-owner/github-repository/pulls/0/reviews')
+        .reply(200, []);
 
     // when
     await githubService.getPullRequests('Tech Review Needed');
 
     // then
-    sinon.assert.calledWith(axios.get, expectedUrl);
+    // this will fail if nock are not used
```

Comment:
suggestion : Utiliser le scope nock https://github.com/nock/nock#expectations
```suggestion
    expect(scope.isDone()); 
```